website/Contribution/CONTRIBUTING.rst

```diff
@@ -607,7 +607,7 @@ Idioms
 * For Python 3 (Odoo >= 11.0), no need for utf-8 coding line as this is
   implicit.
 * Prefer `%` over `.format()`, prefer `%(varname)` instead of positional.
-  This is better for translation and clarity.
+  This is better for translation and security.
```

Comment:
It would be nice to have a link to some source that explains why it is a security issue.